modules/seqexec/server/src/main/scala/seqexec/server/altair/AltairControllerEpics.scala

```diff
@@ -119,15 +119,15 @@ object AltairControllerEpics extends AltairController[IO] {
     reasons: Set[Gaos.ResumeCondition]): Option[IO[Unit]] = {
     val offsets = reasons.collectFirst { case OffsetReached(o) => o }
     val newPosOk = offsets.forall(v => newPosInRange(newPosition(starPos)(v)))
-    val guideOk = reasons.contains(GaosStarOn)
+    val guideOk = reasons.contains(GaosGuideOn)
```

Comment:
I renamed because in LGS+OI and LGS+P1 there is no guide star, so the old name didn't make sense.